internal/scm/utils/slice.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+func SliceContainsEntryFromSlice(slice1 []string, slice2 []string) bool {
```

Comment:
`util` is too generic of a package name

https://go.dev/blog/package-names#bad-package-names

This could instead be placed directly under `scm`. Since we now use it for multiple different packages, I think it also makes sense to name it after what we want to archive,

So in total:
* Move to `scm/repository.go`
* Rename to something like `RepoContainsTopic` (or similar)

Reply:
Mhm but that function does something generic, why call it by something specific? 
But its changed now.